scripts/jenkins/cloud/manual/deployer-entry-point.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -i
+
+: ${git_user_name:=Unknown User}
+: ${git_user_email:=unknown@unknown.org}
+: ${python:=python2}
+
+configure_git() {
+    git config --global user.name "${git_user_name}"
+    git config --global user.email "${git_user_email}"
+}
+
+set_up_venv() {
+    ln -s "/opt/ansible-venv-${python}" /opt/ansible-venv
+}
+
+set -u -x
+
+while (( $# > 0 )); do
+    case $1 in
+        "--git-user-name")
+            shift
+            git_user_name=$1
+            shift
+            ;;
+        "--git-user-email")
+            shift
+            git_user_email=$1
+            shift
+            ;;
+        "--python2"|"--python3")
+            python=${1#--}
+            shift
+            ;;
+        *)
+            echo "Unknown argument $1, terminating parsing"
+            break
+            ;;
+    esac
+done
+
+configure_git
+set_up_venv
+
+/opt/ansible-venv-python3/bin/python /opt/bin/copy_clouds.py \
```

Comment:
This section fails with "directory exists" if it is run a second time.  
For example when someone logs off container and attaches again and runs /deployer-entry-point.sh